packages/ove-asset-manager/README.md

```diff
@@ -1,7 +1,112 @@
-To run this Service you must add Object store credentials into the Appsettings.json file as follows
+# OVE Asset Manager Services
 
-  "s3Client": {
-    "AccessKey": "secret",
+This collection of microservices provides end users with a simple way of managing and processing content to be displayed using OVE. 
+
+## Asset Manager
+
+The core service is the **Asset Manager**. This enables media and content to be uploaded to an **S3** compatible **object store** and records metadata about its processing in an **SQL database**. A full user friendly website is provided to interact with the Asset Manager. A full RESTful API is provided for advanced users.  
+
+## Asset Services
+
+A variety of other Asset Services are provided to process assets into a form advantageous for display using OVE. These services are generally headless and will automatically register themselves with an Asset Manager via its API. They will then generally automatically process assets which require their services. For example the [**Image Tile Service**](https://github.com/ove/ove-asset-services/blob/master/packages/ove-service-imagetiles/README.MD) will create a Deep Zoom image (.dzi file) and corresponding tileset from each image asset. 

```

Comment:
Change `.MD` -> `.md` so the link works:

```suggestion
A variety of other Asset Services are provided to process assets into a form advantageous for display using OVE. These services are generally headless and will automatically register themselves with an Asset Manager via its API. They will then generally automatically process assets which require their services. For example the [**Image Tile Service**](https://github.com/ove/ove-asset-services/blob/master/packages/ove-service-imagetiles/README.md) will create a Deep Zoom image (.dzi file) and corresponding tileset from each image asset. 
```

Reply:
This doesn't seem to have been fixed: the link is still to `README.MD` rather than `README.md`

Reply:
updated to use relative paths.